hadoop-ozone/dev-support/checks/_mvn_unit_report.sh

```diff
@@ -37,7 +37,7 @@ find "." -not -path '*/iteration*' -name 'TEST*.xml' -print0 \
 if [[ "${CHECK:-unit}" == "integration" ]]; then
   find hadoop-ozone/integration-test -not -path '*/iteration*' -name '*-output.txt' -print0 \
       | xargs -n1 -0 "grep" -l -E "not closed properly|was not shutdown properly" \
-      | awk -F/ '{sub("-output.txt",""); print $NF}' \
+      | awk -F/ '{sub("-output.txt",""); print $NF ": Memory Leak detected in test, Failing."}' \
```

Comment:
Let's not add this.  Output is expected to contain test class names, extra description will break processing below.

Reply:
Can we append a message to end of the TestName-output.txt instead of summary.txt  . I am insisting on this because it is hard to understand why the CI failed in such situations

Reply:
I'll check: HDDS-11551.